Common/UnitDefinitions/ElectricPotentialAc.json

```diff
@@ -1,7 +1,8 @@
 {
   "Name": "ElectricPotentialAc",
   "BaseUnit": "VoltAc",
-  "XmlDocSummary": "The Electric Potential of a system known to use Alternating Current.",
+  "XmlDocSummary": "If you want to map more parameters into the ElectricPotential class (volts RMS, phase angle, etc.), just use a named tuple.",
+  "ObsoleteText": "This class is redundant. Please use ElectricPotential instead.",
```

Comment:
I would keep the XmlDocSummary as before, and merge the new text into the ObsoleteText instead - since this is what's thrown in your face by intellisense and build warnings.

E.g. `ElectricPotentialAc was renamed to ElectricPotential, and will be removed in a later major version. If you want to map more parameters into the ElectricPotential class (volts RMS, phase angle, etc.), create your own wrapper type such as a record or named tuple.`

Reply:
This text about named tuple would be helpful for the new types as well, not just the obsoleted ones.